test/api/auth_test.rb

```diff
@@ -0,0 +1,98 @@
+require "minitest/autorun"
+require_relative "../test_helper"
+
+describe "auth API" do
+  before do
+    db.clean
+  end
+
+  describe "POST /api/account" do
+    before do
+      skip_if_impl_in %w(elixir_phoenix node_express node_hapi ruby_sinatra)
+    end
+
+    it "registers a new user account" do
+      response = post("/api/account", {
+        data: [{
+          type: "accounts",
+          attributes: {
+            name: "Micah Geisel",
+            email: "micah@botandrose.com",
+            password: "omgponies",
+          }
+        }]
+      })
+
+      # we can't stub time, so pull out timestamps and assert on format :(
+      json_body = response.json_body
+      created_at = json_body["data"][0]["attributes"].delete("created_at")
+      updated_at = json_body["data"][0]["attributes"].delete("updated_at")
+      created_at.must_match /\A\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z\Z/
+      updated_at.must_match /\A\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z\Z/
+
+      json_body.must_equal_json({
+        data: [{
+          type: "accounts",
+          id: "1",
+          attributes: {
+            name: "Micah Geisel",
+            email: "micah@botandrose.com",
+          },
+        }]
+      })
+      response.code.must_equal 201
+
+      response = post("/api/auth", {
+        data: [{
+          type: "auth",
+          attributes: {
+            email: "micah@botandrose.com",
+            password: "badpassword",
+          },
+        }]
+      })
+      response.json_body.must_equal_json({
+        errors: [{
+          title: "Invalid authentication credentials",
+        }]
+      })
+      response.code.must_equal 403
```

Comment:
IMO this should be a 401.  403 means you have successfully authenticated (i.e. the server know who you are) but you are forbidden from the requested resource.